kpi/deployment_backends/kc_reader/shadow_models.py

```diff
@@ -0,0 +1,78 @@
+from django.contrib.auth.models import User
+from django.db import models
+from django.db import ProgrammingError
+
+
+class ReadOnlyModelError(ValueError):
+    pass
+
+
+class _ReadOnlyModel(models.Model):
+    class Meta:
+        abstract = True
+
+    def save(self, *args, **kwargs):
+        raise ReadOnlyModelError('Cannot save read-only-model')
+
+    def delete(self, *args, **kwargs):
+        raise ReadOnlyModelError('Cannot delete read-only-model')
+
+
+class LazyModelGroup:
```

Comment:
This is the best way I could find to prevent `manage.py makemigrations` from recognizing these models.